dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -183,14 +189,30 @@ JarLinkageReport generateLinkageReport(Path jarPath, SymbolReferenceSet symbolRe
     return linkageErrors;
   }
 
+  @VisibleForTesting
+  Optional<StaticLinkageError<MethodSymbolReference>> checkLinkageErrorMissingMethodAt(
+      MethodSymbolReference reference) {
+    return checkLinkageErrorMethodAt(reference, false);
+  }
+
+  @VisibleForTesting
+  Optional<StaticLinkageError<MethodSymbolReference>> checkLinkageErrorMissingInterfaceMethodAt(
```

Comment:
As above, if `MethodSymbolReference` encoded whether it was an interface method reference or not, would you need this duplication?